src/isValidRequest.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable max-len */
+
+function isValidReq(text, searchProperty, searchParam) {
```

Comment:
It's better to use more descriptive function name like 'isValidRequest' instead of 'isValidReq'.